app/views/bundle_contexts/_yaml_form.erb

```diff
@@ -0,0 +1,65 @@
+<h3>Complete the YAML form below</h3>
+
+<%= simple_form_for(@bundle_context) do |form| %>
+
+    <div class="row">
+        <div class="col-5">
+            <%= form.input :project_name %>
+        </div>
+    </div>
+
+    <div class="row">
```

Comment:
This is much cleaner than the previous design, `simple_form` makes this view more of template than needing to add conditional logic that the previous design required.